packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -1867,6 +1918,112 @@ export default class Meeting extends StatelessWebexPlugin {
     );
   }
 
+  /**
+   * Set up the listeners for captions
+   * @returns {undefined}
+   * @private
+   * @memberof Meeting
+   */
+  private setUpVoiceaListener() {
+    // @ts-ignore
+    this.webex.internal.voicea.on(
+      VOICEAEVENTS.VOICEA_ANNOUNCEMENT,
+      (payload: Transcription['languageOptions']) => {
+        this.transcription.languageOptions = payload;
+        Trigger.trigger(
+          this,
+          {
+            file: 'meeting/index',
+            function: 'setUpVoiceaListener',
+          },
+          EVENT_TRIGGERS.MEETING_STARTED_RECEIVING_TRANSCRIPTION,
+          payload
+        );
+      }
+    );
+
+    // @ts-ignore
+    this.webex.internal.voicea.on(VOICEAEVENTS.CAPTION_LANGUAGE_UPDATE, (payload) => {
+      const {statusCode} = payload;
+
+      if (statusCode === 200) {
+        this.transcription.languageOptions = {
+          ...this.transcription.languageOptions,
+          currentCaptionLanguage:
+            this.transcription.languageOptions.requestedCaptionLanguage ?? LANGUAGE_ENGLISH,
+        };
+        Trigger.trigger(
+          this,
+          {
+            file: 'meeting/index',
+            function: 'setUpVoiceaListener',
+          },
+          EVENT_TRIGGERS.MEETING_CAPTION_LANGUAGE_CHANGED,
+          {languageCode: this.transcription.languageOptions.currentCaptionLanguage}
+        );
+      } else {
+        // TODO Handle Status Code and alert - SPARK-370923
```

Comment:
This TODO is originally from Web Client. Looks like Arun has copied and pasted it here. I shall simply remove this and throw an error in case of a failure. Will check and update you @sreenara

Reply:
This TODO is invalid for us @sreenara. We throw error in all possible scenarios. This TODO is for Web Client to show UI Alerts based on status code.